Core/Src/CAN/inverter_can.c

```diff
@@ -59,3 +62,23 @@ void CAN_inverter_receive_callback(FDCAN_HandleTypeDef* hfdcan, uint32_t RxFifo0
 		}
 	}
 }
+
+
+
+void CAN_inverter_error_callback(FDCAN_HandleTypeDef *hfdcan, uint32_t ErrorStatusITs){
+    if(ErrorStatusITs |= FDCAN_IT_BUS_OFF){
```

Comment:
socorro, eu sugeri errado pra vc kkkkkkkkk
na vdd era:
```suggestion
    if(ErrorStatusITs | FDCAN_IT_BUS_OFF){
```
`|=` é ou-atribuição. Funciona, mas n é isso q vc quer rs